custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -112,20 +112,31 @@ def _reg_arg_target_feature_and_filename(*parsers):
                 ArgumentsOptions.TARGET,
                 type=str,
                 required=False,
-                help="Which column to use as the target. Argument is mutually exclusive with {}".format(
-                    ArgumentsOptions.TARGET_FILENAME
+                help="Which column to use as the target. Argument is mutually exclusive with {} and {}".format(
+                    ArgumentsOptions.TARGET_FILENAME, ArgumentsOptions.UNSUPERVISED
```

Comment:
you can make arguments mutually exclusive using argparse https://stackoverflow.com/questions/17909294/python-argparse-mutual-exclusive-group

Reply:
I think these three are mutually exclusive as things stand? all three are added to "group" which is defined: `group = parser.add_mutually_exclusive_group(required=True)` on line 110

Reply:
got it, I didn't notice that.